Cargo.toml

```diff
@@ -84,6 +84,10 @@ features = ["attributes"]
 version = "0.4"
 optional = true
 
+[dependencies.time]
+version = "0.3"
```

Comment:
This seems to disagree with the title of the PR (0.3 vs. 0.4)

Reply:
True.